tests/conftest.py

```diff
@@ -7088,7 +7091,8 @@ def teardown():
         Cleans up the CNV workloads
 
         """
-        for cnv_wl in cnv_workloads:
+        # Iterating from end so that restored VMs are deleted before source
```

Comment:
Is deleting the restored VMs before the source causing any problems? At least in PVCs, IIRC, I don't think there will be any issues. The parent PVC can be deleted even with the restored PVC in use

Reply:
I will take care of this in my next immediate PR.

Reply:
Based on avadhoot's comments, I will merge this PR as it's required urgently and low priority review comments will be handled in a followup PR